guides/v2.1/ui_comp_guide/concepts/ui_comp_template_literals.md

```diff
@@ -21,7 +21,7 @@ Template literals allow UI Components to easily assign dynamic values to class p
 
 ### The `defaults` Class Property
 
-UI Components are [associated with Javascript classes](http://devdocs.magento.com/guides/v2.1/ui_comp_guide/concepts/ui_comp_uiclass_concept.html) to handle behavior on the client side. These should extend one of the core classes to provide a base level of functionality. Inside the child class, a `defaults` property can be provided.
+UI Components are [associated with Javascript classes](http://devdocs.magento.com/guides/v2.2/ui_comp_guide/concepts/ui_comp_uiclass_concept.html) to handle behavior on the client side. These should extend one of the core classes to provide a base level of functionality. Inside the child class, a `defaults` property can be provided.
```

Comment:
@ryantfowler, why are you linking to 2.2 from a 2.1 topic when the 2.1 topic is legitimate?

Reply:
@jeff-matthews hi, I apologize if I have made a mistake :(.